WcaOnRails/app/helpers/registrations_helper.rb

```diff
@@ -1,3 +1,14 @@
 # frozen_string_literal: true
 module RegistrationsHelper
+  def fees_hint_and_context(connected_stripe_account_id, fees_to_pay)
+    if connected_stripe_account_id
+      if fees_to_pay <= 0
+        [t('registrations.entry_fees_fully_paid', paid: paid_entry_fees), "success"]
```

Comment:
Someone will probably notice this variable is undefined, I'll fix it and smoke test the register form!